lib/yauzl-ts/inputProcessors.ts

```diff
@@ -128,25 +125,24 @@ export function fromBuffer(
   options.autoClose = false
 
   // limit the max chunk size. see https://github.com/thejoshwolfe/yauzl/issues/87
-  // eslint-disable-next-line @typescript-eslint/no-unsafe-call,@typescript-eslint/no-unsafe-member-access
-  const reader = fd_slicer.createFromBuffer(buffer, { maxChunkSize: 0x10000 })
+  const reader = createFromBuffer(buffer, { maxChunkSize: 0x10000 })
   fromRandomAccessReader(reader, buffer.length, options, callback)
 }
 
-export function fromRandomAccessReader<TReader extends RandomAccessReader>(
+export function fromRandomAccessReader<TReader extends IRandomAccessReader>(
```

Comment:
TypeScript naming conventions explicitly advise against prefixing interfaces

Reply:
Any suggestion of naming?